frontend/src/components/InfoPanel.jsx

```diff
@@ -5,6 +5,7 @@ import Col from "react-bootstrap/Col";
 import Container from "react-bootstrap/Container";
 import Row from "react-bootstrap/Row";
 import Histogram from "./Histogram";
+import MyDatesPicker from "./MyDatesPicker";
```

Comment:
Can we change this name to either `DatesPicker` or something like `CounterfactualDatesPicker`?

Reply:
Yeah, this is being refactored in PR #77 .